From 189c84569695079c798409473c746452c2fbfa2f Mon Sep 17 00:00:00 2001
From: B00625 <B00625@freesacle.com>
Date: Fri, 11 Jun 2010 16:17:42 +0800
Subject: [PATCH 05/39] Delete the SMS in ME memory each time SIM card is
 ready. avoid ME memory full as it can only hold 23
 SMSs Signed-Off-By: Yinhaiting <b00625@freescale.com>

---
 reference-ril/reference-ril.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/reference-ril/reference-ril.c b/reference-ril/reference-ril.c
index 32a1fe7..215afa0 100755
--- a/reference-ril/reference-ril.c
+++ b/reference-ril/reference-ril.c
@@ -374,6 +374,7 @@ static void onSIMReady()
      */
 at_send_command("AT+CNMI=1,2,2,1,1", NULL);
 at_send_command("AT+CPMS=\"ME\",\"ME\",\"ME\"",NULL);
+at_send_command("AT+CMGD=1,4",NULL);
 }
 
 static void requestRadioPower(void *data, size_t datalen, RIL_Token t)
-- 
1.8.0

